src/fusor/fusor.py

```diff
@@ -52,6 +52,9 @@
 _logger = logging.getLogger(__name__)
 
 
+CURIE_REGEX = r"^\w[^:]*:.+$"
```

Comment:
Gene Normalizer defines this in schemas. I'd consider using this rather than redefining.